services/src/main/webapp/WEB-INF/web.xml

```diff
@@ -58,6 +58,10 @@
             <web-resource-name>Public</web-resource-name>
             <url-pattern>/ArquillianServletRunner/*</url-pattern>
         </web-resource-collection>
+        <web-resource-collection>
```

Comment:
You are right, we need to fix this. Unfortunately there is no standard way of doing it, so we basically have 2 options:

1) Send token as GET parameter in URL (ugly a probably not as secure as 2 )
2) After opening connection, send JSON object with token

(based on http://lists.jboss.org/pipermail/keycloak-user/2015-August/002815.html )

I have found hawkular API for it:
https://github.com/hawkular/hawkular-accounts/tree/master/websocket-api

I will test it and find out if we could use it. I don't like this statement:

> The intended usage is to have your Server Web Socket endpoints as a separate deployment, without Keycloak’s configuration at web.xml.

but we will see if it could be combined.